invenio_oauthclient/views/client.py

```diff
@@ -126,7 +127,17 @@ def authorized(remote_app=None):
            not(current_app.debug or current_app.testing)):
             abort(403)
 
-    return current_oauthclient.handlers[remote_app]()
+    try:
+        handler = current_oauthclient.handlers[remote_app]()
+    except OAuthException as e:
+        if e.type == 'invalid_response':
+            flash('Code returned was invalid',
+                  category='danger')
+            return redirect('/')
```

Comment:
Maybe we should consider `abort` instead of `redirect`. It looks like an invalid flow ...